@here/harp-mapview/lib/MapView.ts

```diff
@@ -642,14 +629,42 @@ export const MapViewDefaults = {
         typeof window !== "undefined" && window.devicePixelRatio !== undefined
             ? window.devicePixelRatio
             : 1.0,
-    geoCenter: new GeoCoordinates(25, 0, 30000000),
     target: new GeoCoordinates(25, 0),
     zoomLevel: 5,
     tilt: 0,
     heading: 0,
     theme: {}
 };
 
+/**
+ * Parameters for MapView.lookAt
+ */
+export interface LookAtParams {
+    /**
+     * Target/look at point of the MapView.
+     * @note If the given point is not on the ground (altitude != 0) [[MapView]] will do a
+     * raycasting internally to find a target on the ground.
+     * As consequence [[MapView.target]] and [[MapView.zoomLevel]] will nit match the values
+     * that were passed into the [[MapView.lookAt]] method.
+     */
```

Comment:
Add notes on defaults to all params if used in context of `MapViewOptions` and when used as `lookAt`.

Reply:
done